src/app/shared/palette/order-summary/fee.model.ts

```diff
@@ -0,0 +1,5 @@
+export class Fee {
+    FeeCode: string;
```

Comment:
how come fields are upper case?

Reply:
This is the case details data contract we have with services.